### PR TITLE
Update AR.yml

### PR DESCRIPTION
**Pull request by gabriel-seow**

### CODE REVIEW

File: public/data/travel/AR.yml [hunk 1]
```diff
@@ -1,23 +1,57 @@
 ---
 reopening:
-  '2020-09-01': Commercial flights may [reopen on September 1, 2020](https://www.garda.com/crisis24/news-alerts/361251/argentina-authorities-ease-covid-19-restrictions-in-buenos-aires-metropolitan-area-from-july-20-update-21).
+  '2020-09-01': Commercial flights may [reopen on September 20, 2020](https://secretsofbuenosaires.com/argentina-travel-update-when-will-tourism-resume/).
```
Comment (jannwong) [inline public/data/travel/AR.yml:3]: date needs to change to 2020-09-20

File: public/data/travel/AR.yml [hunk 2]
```diff
@@ -1,23 +1,57 @@
...
+  interstate_commercial_flights: 'no'
+  interstate_commercial_flights_source: https://simpleflying.com/argentina-flight-resumption-delay/
+  interstate_private_land_transportation: 'no'
+  interstate_private_land_transportation_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  interstate_public_transportation: 'no'
+  interstate_public_transportation_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  public_transportation_in_region:
+  public_transportation_in_region_source:
+  mask: 'yes'
+  mask_text: For groups of up to 10 people meeting in the outdoors.
```
Comment (jannwong) [inline public/data/travel/AR.yml:33]: outdoors only? no need to wear masks indoor? or is it in public?

File: public/data/travel/AR.yml [hunk 3]
```diff
@@ -1,23 +1,57 @@
...
+  mask_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  social_gatherings_limit: 'yes'
+  social_gatherings_limit_text: 10 people
+  social_gathering_limit_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  restaurants_open: partial
+  restaurants_capacity_text: Only open in Bueno Aires
```
Comment (jannwong) [inline public/data/travel/AR.yml:39]: need full stop at the of sentence

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comments

File: public/data/travel/AR.yml (hunk 2 above)
Comment (gabriel-seow) [reply public/data/travel/AR.yml:33]: Only outdoors!

Comment (jannwong) [reply public/data/travel/AR.yml:33]: what a weird rule...indoor is more risky. lol

File: public/data/travel/AR.yml [hunk 4]
```diff
@@ -1,23 +1,57 @@
...
+  interstate_private_land_transportation_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  interstate_public_transportation: 'no'
+  interstate_public_transportation_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  public_transportation_in_region:
+  public_transportation_in_region_source:
+  mask: 'yes'
+  mask_text: For groups of up to 10 people meeting in the outdoors.
+  mask_source: https://www.garda.com/crisis24/news-alerts/373801/argentina-authorities-extend-covid-19-measures-until-september-20-update-25
+  social_gatherings_limit: 'yes'
+  social_gatherings_limit_text: 10 people
```
Comment (jannwong) [inline public/data/travel/AR.yml:36]: need a full sentence and with full stop

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comment

**Review by jannwong — APPROVED** [review]

**Pull request merged**